scripts/floater_convert

```diff
@@ -36,6 +36,9 @@ parser.add_argument('--output_prefix', default='float_trajectories', metavar='OP
 parser.add_argument('--ref_time', default=None, metavar='RT',
                     help='reference time, format: YYYY-MM-DD')
 
+parser.add_argument('--pkl_path', default=None, metavar='PP',
+                    help='path to the pickle file')
+
```

Comment:
be more verbose here: `path the the pickle file of the FloatSet used to generate initial positions`.